brianlong/www.validators.app
# 186282710 stake explorer tweaks

#### What's this PR do?
- change routes to stake-explorer
- add validator to explorer details
- add validator to explorer index
- update docs

#### How should this be manually tested?
- check index and detail pages for validator presence
- check api docs
- check api endpoint

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186282710)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
